packages/backend-for-frontend/src/routers/supportRouter.ts

```diff
@@ -34,14 +34,11 @@ const supportRouter = (
   );
 
   supportRouter.post("/session/saml2/tokens", async (req, res) => {
-    const { tenantId, saml2 } = req.params;
     const ctx = fromBffAppContext(req.ctx, req.headers);
 
     try {
-      const samlDecoded = Buffer.from(saml2, "base64").toString();
```

Comment:
Moving this inside the service function after the first log, we usually don't perform any operation in the router